### PR TITLE
Updated Identification .arg message to reflect changes in Arrivals

### PR DESCRIPTION
**Pull request by Reece-Carruthers**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -1094,7 +1094,7 @@ transport.transportMeans.departure.inlandMode.error.required = Select which inla
 transport.transportMeans.departure.identification.title = Which identification do you want to use for this vehicle?
 transport.transportMeans.departure.identification.heading = Which identification do you want to use for this vehicle?
 transport.transportMeans.departure.identification.seaGoingVehicle = Name of a sea-going vehicle
```
Comment (kevincoatsworth) [inline conf/messages.en:1096]: Could you change this to vessel as well?

Comment (Reece-Carruthers) [reply conf/messages.en:1096]: Can do good spot

File: app/models/transport/transportMeans/departure/Identification.scala [hunk 2]
```diff
@@ -27,7 +27,7 @@ sealed trait Identification {
 
 object Identification extends RadioModelU[Identification] {
 
-  case object SeaGoingVehicle extends WithName("seaGoingVehicle") with Identification {
+  case object SeaGoingVehicle extends WithName("seaGoingVessel") with Identification {
```
Comment (kevincoatsworth) [inline app/models/transport/transportMeans/departure/Identification.scala:30]: Can you change the object name too?

File: test/models/transport/transportMeans/departure/IdentificationSpec.scala [hunk 3]
```diff
@@ -63,7 +63,7 @@ class IdentificationSpec extends SpecBase with ScalaCheckPropertyChecks with Gen
 
       "when sea going vehicle" in {
         val result = SeaGoingVehicle.arg
```
Comment (kevincoatsworth) [inline test/models/transport/transportMeans/departure/IdentificationSpec.scala:65]: Add change the object name here too

**Review by kevincoatsworth — APPROVED** [review]

**Pull request merged**